libqtile/widget/windowname.py

```diff
@@ -56,14 +66,17 @@ def update(self, *args):
             w = self.qtile.current_screen.group.current_window
         else:
             w = self.bar.screen.group.current_window
-        state = ''
+        state = ""
         if self.show_state and w is not None:
             if w.maximized:
-                state = '[] '
+                state = "[] "
             elif w.minimized:
-                state = '_ '
+                state = "_ "
             elif w.floating:
-                state = 'V '
-        unescaped = "%s%s" % (state, w.name if w and w.name else " ")
+                state = "V "
+        unescaped = "%s%s" % (
+            state,
+            w.name if w and w.name else self.empty_group_string,
```

Comment:
Can you drop all the other formatting changes and just do this?

Reply:
I believe I just fixed it. Let me know if I missed anything!